libs/fileset/file.go

```diff
@@ -19,3 +56,21 @@ func (f File) Modified() (ts time.Time) {
 	}
 	return info.ModTime()
 }
+
+func (f *File) IsNotebook() (bool, error) {
```

Comment:
Even though the function has a pointer receiver, the function is sometimes called on a value, which means the result isn't written back to the original value (e.g. when it is the element in a for loop).